appveyor.yml

```diff
@@ -12,7 +12,7 @@ environment:
     CONDA_DEPENDENCIES: >
       rope pyflakes sphinx pygments pylint pycodestyle psutil nbconvert
       qtawesome pickleshare pyzmq chardet mock pandas pytest
-      pytest-cov numpydoc scipy pillow qtconsole matplotlib jedi
+      pytest-cov numpydoc scipy pillow qtconsole matplotlib jedi pywin32
```

Comment:
@jnsebgosselin Why are you adding pywin32 as a test dependency? According to its author, packages shouldn't depend on it: https://github.com/spyder-ide/pywinpty/issues/52#issuecomment-324114565

Reply:
Because it is needed for my test to pass. Otherwise, it hangs due to this [warning message dialog](https://github.com/spyder-ide/spyder/blob/master/spyder/utils/environ.py#L95). @jitseniesen was right, my mocking was working as expected, it was this other message that was blocking the test.

Reply:
So I don't know how I should handle that. If I can't use pywin32, then I think I can't really test this feature, unless I'm able to logout/login of the VM during the test.

Reply:
I don't have a problem with this, given that it's currently required by the @jnsebgosselin is trying to test.

Of course, we should look at removing this in the future.

Reply:
Let's see what @ccordoba12 thinks!